docs/pom.xml

```diff
@@ -104,6 +104,171 @@
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-appconf-configprops</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>java</goal>
+                </goals>
+                <configuration>
+                  <arguments>
+                    <argument>${project.basedir}/src/main/asciidoc/_configprops_appconf.adoc</argument>
+                    <argument>spring.cloud.azure.appconfiguration.*</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-cosmos-configprops</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>java</goal>
+                </goals>
+                <configuration>
+                  <arguments>
+                    <argument>${project.basedir}/src/main/asciidoc/_configprops_cosmos.adoc</argument>
+                    <argument>spring.cloud.azure.cosmos.*</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-eventhubs-configprops</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>java</goal>
+                </goals>
+                <configuration>
+                  <arguments>
+                    <argument>${project.basedir}/src/main/asciidoc/_configprops_eventhubs.adoc</argument>
+                    <argument>spring.cloud.azure.eventhubs.*</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-kv-secret-configprops</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>java</goal>
+                </goals>
+                <configuration>
+                  <arguments>
+                    <argument>${project.basedir}/src/main/asciidoc/_configprops_kv_secret.adoc</argument>
+                    <argument>spring.cloud.azure.keyvault.secret.*</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-kv-certificate-configprops</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>java</goal>
+                </goals>
+                <configuration>
+                  <arguments>
+                    <argument>${project.basedir}/src/main/asciidoc/_configprops_kv_certificate.adoc</argument>
+                    <argument>spring.cloud.azure.keyvault.certificate.*</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-servicebus-configprops</id>
```

Comment:
need we add configuration properties for jms as well?

Reply:
good point

Reply:
added